### DISCUSSION TITLE
Quizzes

### DISCUSSION THREAD

**Discussion by akanz1** [OpenMined/courses#74] [Feedback] [answered]
Hey, OpenMined Team, I'm really enjoying the first course so far! 👍 

A suggestion i have is regarding the quizzes. Currently, the first radio button we click on is directly accepted as the users answer, without requiring any confirmation (i.e. clicking on "next" or "submit"), often times resulting in unintended selections.

**Comment by yemikifouly** [comment, marked as answer]
Hello Andreas! 👋 

I am really glad you are enjoying the course so far! 🙂 It is a very interesting point that you bring up. I will pass it as a feature request to our Web Developer. Thanks very much for suggesting it! And keep the feedback coming. 🙂
